documentation/planning.md

```diff
@@ -50,18 +50,33 @@ So each LED per zone will get a number from 0 to n, where n is the last LED in t
 
 ![LED Chain](media/planning/led-chain.png)
 
-Now you have to make an important choice between three options for each part of the car.
-They are all fully supported by TesLight.
+### Luminant Types
+Currently there are three different types of luminants available that you can choose from. They are all fully supported by TesLight, but not each luminant is suitable for each installation location.
 
-- Use a basic WS2812 LED strip
-- Use a fiber tube and 2 WS2812 LEDs as light injectors
-- Use a light bar which looks cleaner than a fiber tube and allows awesome animations
+#### Basic WS2812 LED strip 
+This is the most widely availble type of luminant - a series of WS2812 LEDs in mounted in series on a thin, flexible PCB strip. These strips are available in various lenghts, widths and density (count of LEDs per meter). 
+Possible installation locations for the LED strip are the dashboard and the foot wells. A density of around 60 LEDs/meter is a good compromise between cost/power consumption and good looking light (effects).
+Benefits of the LED strip are that it is easy to obtain in various configurations and since many LEDs are distributed across the strip it can be used for many light effects.
+The main drawback is that it can only be installed in locations where it is not directly visible.
+
+#### Fiber tube with WS2812 LEDs as light injectors
+This type of luminant is a small flexible tube with a tab on it's side. WS2812 LEDs can be placed on one or (better) both ends of the tube to inject light into the tube and make it glow (fairly) evenly along its entire length.
+The fiber tube is suitable for all installation locations in the Tesla Model 3/Y with the exception of the footwell lighting. As of now it is also the only lumimant that can be installed in the 2021 or later center console of Model 3/Y and around the seats.
```

Comment:
I do not see why light bars should not be used around the seats as well

Reply:
I don't think this would work because they can only be bent in one dimension (perpendicular to the plane of the bar) but for the corners at the top of the seats (or also the center console) they would need to be bent outward and parallel to the plane of the bar. (Yeah, that's a shitty explanation but I don't know how to better phrase it)